message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1708,5 +1708,19 @@
       <field type="float" name="min_distance" units="m">Minimum distance the sensor can measure.</field>
       <field type="float" name="max_distance" units="m">Maximum distance the sensor can measure.</field>
     </message>
+    <message id="11038" name="WATER_DEPTH">
+      <description>Water depth</description>
+      <field type="uint32_t" name="time_boot_ms" units="ms">Timestamp (time since system boot)</field>
+      <field type="uint8_t" name="id" instance="true">Onboard ID of the sensor</field>
+      <field type="uint8_t" name="healthy">Sensor data healthy (0=unhealthy, 1=healthy)</field>
```

Comment:
This could be `state` instead.  We use that in winch and in battery stuff.

We can retrofit an enum later.

Reply:
you may be right but I suspect it'll never be changed so I might just leave it as healthy..